test/functional/terminal/cursor_spec.lua

```diff
@@ -93,39 +93,44 @@ describe(':terminal cursor', function()
       show_cursor()
       screen:expect([[
         tty ready                                         |
-        {1: }                                                 |
+        ^                                                  |
                                                           |*4
         {3:-- TERMINAL --}                                    |
       ]])
       -- same for when the terminal is unfocused
       feed('<c-\\><c-n>')
       hide_cursor()
-      screen:expect([[
+      screen:expect({
+        grid = [[
         tty ready                                         |
         ^                                                  |
                                                           |*5
-      ]])
+      ]],
+        unchanged = true,
+      })
       show_cursor()
-      screen:expect([[
+      screen:expect({
+        grid = [[
         tty ready                                         |
-        {2:^ }                                                 |
+        ^                                                  |
                                                           |*5
-      ]])
+      ]],
+        unchanged = true,
+      })
     end)
   end)
 end)
 
-describe('cursor with customized highlighting', function()
+-- Pending until screen.lua supports coloring the cursor
+pending('cursor with customized highlighting', function()
```

Comment:
`TermCursor` highlights work but AFAICT screen.lua doesn't handle coloring the cursor based on hl groups from `'guicursor'`. I tried implementing this but ran out of time tonight. Will try again later.

Reply:
I'm wondering if we should just remove this test. The terminal cursor color is set in `'guicursor'` now, so is colored by the UI. So this is a UI test now, not a Nvim core test, and we already have a test for `'guicolor'` in `ui/cursor_spec.lua`.